### DISCUSSION TITLE
Notifica accesso da posizione/device sconosciuti

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3722] [Ideas]
Buongiorno,
compilavo una checklist giratami dal nostro avvocato per la stesura della privacy policy.
Tra le varie domande relative alla sicurezza viene chiesto: _L'utente viene avvisato quando viene effettuato un accesso da un device sconosciuto?_
Il mio sito attualmente è accessibile solo da rete locale quindi non ho modo di verificarlo.
Nel caso non fosse presente questa feature, potrebbe essere una implementazione sensata?

**Comment by evilaliv3** [comment]
Cosa si intende per device sconosciuti?

A momento la risposta è certamente no in quanto il sistema non effettua alcuna registrazine dei metadati di accesso.

La proposta i feature è certamente sensata. Provvedo ad aprire un ticket per tenerla in considerazione e vedere cosa ne pensa la community internazionale.
A livello teorico si potrebbe certamente prevedere di farlo ed inviare eventualmente una email all'utente con un link da cliccarsi per abilitare il dispositivo (come fa ad esempio google sui telefoni); A livello pratico forse però complicherebbe di molto l'usabilità della piattaforma.

Voi avreste mdo di supportarne lo sviluppo?

**Comment by maelaborazioni** [comment]
Sì, esatto si tratterebbe di fare un po' quello che fa Google. Però per farlo Google ti dice da dove arriva la connessione quindi credo che non sia compatibile con le vostre scelte tecniche orientate alle privacy (che peraltro condivido). Non so se esiste un modo per poterla implementare in ogni caso. Mi confronto con i colleghi per capire cosa ne pensano.
Intanto grazie per la risposta!
